akka-discovery/src/test/scala/akka/discovery/LookupSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2017-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.discovery
+
+import org.scalatest.{ Matchers, OptionValues, WordSpec }
+
+class LookupSpec extends WordSpec with Matchers with OptionValues {
+
+  // SRV strings with invalid domain names
+  // should fail to build lookups
+  val srvWithInvalidDomainNames = List(
+    "_portName._protocol.service_name.local",
+    "_portName._protocol.servicename,local",
+    "_portName._protocol.servicename.local-",
+    "_portName._protocol.-servicename.local"
+  )
+
+  // No SRV that should result in simple A/AAAA lookups
+  val noSrvLookups = List(
+    "portName.protocol.serviceName.local",
+    "serviceName.local",
+    "_portName.serviceName",
+    "_serviceName.local",
+    "_serviceName,local",
+    "-serviceName.local",
+    "serviceName.local-"
+  )
```

Comment:
Nit: can the closing `)` be on the last line with an entry vs new line?

Reply:
I didn't know there was such a convention on akka project. I personally prefer to separate these things to make it easier to read. The eyes can easily see a dangling parenthesis and know where something finishes. 

But fair enough, I've changed it.